bootstrap/insert_data.py

```
@@ -0,0 +1,120 @@
+    return str(candidate) if candidate.exists() else sys.executable
+
+
+def _run_python_code(label: str, python_exe: str, cwd: Path, code: str) -> None:
+    print(f"[insert_data] {label} 시작 (python={python_exe})")
+    env = os.environ.copy()
+    pythonpath_parts = [str(ROOT)]
+    if existing := env.get("PYTHONPATH"):
+        pythonpath_parts.append(existing)
+    env["PYTHONPATH"] = os.pathsep.join(pythonpath_parts)
+
+    result = subprocess.run([python_exe, "-c", code], cwd=str(cwd), env=env)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)